lib/rubocop/cop/rails/has_many_or_has_one_dependent.rb

```diff
@@ -23,19 +23,20 @@ module Rails
       class HasManyOrHasOneDependent < Cop
         MSG = 'Specify a `:dependent` option.'.freeze
 
-        def_node_matcher :has_many_or_has_one_without_options?, <<-PATTERN
+        def_node_matcher :present_has_many_or_has_one_without_options?,
```

Comment:
I think these changes made the code slightly worse. "Present" is just another prefix, that didn't happen to be in the blacklist. 🙂 

I would rather change the methods to:

- `#association_without_options?`
- `#association_with_options?`
- `#dependent_option?`
- `#present_option?`

Reply:
I got it! Implementation of cop suggested by @pocke takes some time. I'd like to open an independent PR that I got this advice.